### DISCUSSION TITLE
Some parts of text not detecting in video

### DISCUSSION THREAD

**Discussion by karunakk** [mindee/doctr#1282] [General]
When attempting to extract text from videos using Doctr OCR, I encountered an issue where certain text, despite being clearly visible, was not being recognized. Notably, text positioned in the foreground was successfully detected, whereas text in the background remained undetected despite its clear visibility. I'm seeking suggestions to address this matter.

**Comment by felixT2K** [comment]
Hi @karunakk 👋🏼,

We are currently in the stage to train the models without checkpoints: https://github.com/mindee/doctr/issues/969
We have added some more augmentations (pytorch) to make the models more robust. 
The pytorch model trainings will start in around 2 weeks.
If we can reach a significant improvement i think it would be possible to retrain the current available pytorch detection models again after the next release (~november).

@odulcy-mindee
